homeassistant/components/overkiz/number.py

```diff
@@ -225,3 +229,14 @@ async def async_set_native_value(self, value: float) -> None:
         await self.executor.async_execute_command(
             self.entity_description.command, value
         )
+
+    async def async_added_to_hass(self) -> None:
+        """Run when this Entity has been added to HA."""
+        await super().async_added_to_hass()
+
+        if self.entity_description.native_min_key:
+            if state := self.device.states.get(self.entity_description.native_min_key):
```

Comment:
What about an `and` instead of the nested `if` ? It may improve cognitive complexity by removing a level of nesting.
Same thing below.

Reply:
Thank you for your suggestion!. I'll update the code accordingly to reflect this readability improvement.